CMakeLists.txt

```diff
@@ -20,6 +20,25 @@ include(EthToolchains)
 include(EthPolicy)
 eth_policy()
 
+set(SUBMODULE_CONTENTS "${CMAKE_SOURCE_DIR}/deps/fmt")
+file(GLOB SUBMODULE_CONTENTS "${SUBMODULE_CONTENTS}/*")
+if (SUBMODULE_CONTENTS)
```

Comment:
@ekpyron had the idea to only implicitly initialise the submodules, if they where not yet initialised. I think that makes a lot of sense.

Reply:
I'd still recommend to check for the `.git` file in the submodules' subdirectories instead of checking if there's _anything_ in there. It will also only be created if `git submodule update` is run, the `init` step itself doesn't place it.

Reply:
true! that is better!